drivers/include/ng_at86rf2xx.h

```diff
@@ -0,0 +1,359 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    drivers_ng_at86rf2xx AT86RF2xx based drivers
+ * @ingroup     drivers
+ *
+ * This module contains drivers for radio devices in Atmel's AT86RF2xx series.
+ * The driver is aimed to work with all devices of this series.
+ *
+ * @{
+ *
+ * @file
+ * @brief       Interface definition for AT86RF2xx based drivers
+ *
+ * @author      Thomas Eichinger <thomas.eichinger@fu-berlin.de>
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ */
+
+#ifndef NG_AT86RF2XX_H_
+#define NG_AT86RF2XX_H_
+
+#include <stdint.h>
+
+#include "board.h"
+#include "periph/spi.h"
+#include "periph/gpio.h"
+#include "net/ng_netdev.h"
+#include "ng_at86rf2xx.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Maximum possible packet size in byte
+ */
+#define NG_AT86RF2XX_MAX_PKT_LENGTH     (127)
+
+/**
+ * @brief   Default addresses used if the CPUID module is not present
+ * @{
+ */
+#define NG_AT86RF2XX_DEFAULT_ADDR_SHORT (0x0230)
+#define NG_AT86RF2XX_DEFAULT_ADDR_LONG  (0x1122334455667788)
```

Comment:
(also: the long address is as you defined it a globally-unique multicast address).